execution/execution.go

```diff
@@ -37,41 +36,33 @@ func (s Status) String() (r string) {
 
 // Execution stores all informations about executions.
 type Execution struct {
-	ID                string                 `hash:"-"`
+	Hash              string                 `hash:"-"`
 	EventID           string                 `hash:"name:eventID"`
 	Status            Status                 `hash:"-"`
-	Service           *service.Service       `hash:"name:service"`
+	ServiceHash       string                 `hash:"name:serviceHash"`
+	ServiceParentHash string                 `hash:"name:serviceHash"`
```

Comment:
Why `serviceParentHash`?
The goal is to have the previous execution hash. The previous service hash will be fetched from the previous execution hash and anyway not really relevant.
Also, make sure that the hash tag is replaces